tests/pytype_blacklist.txt

```diff
@@ -8,39 +8,9 @@ stdlib/2/typing.pyi
 stdlib/3/builtins.pyi
 stdlib/3/typing.pyi
 
-# pytype doesn't yet support "# type: ignore" after decorators, sometimes
-# used when overriding methods, especially those inherited from container
-# abstract base classes:
-stdlib/3/array.pyi
-
-# Because of 'from . import path':
-stdlib/2/os/__init__.pyi
-stdlib/3/os/__init__.pyi
-
-# We've not been running the pytype tests under Python 3, for quite a while.
-# The below are files that have regressed and need fixing:
+# pytype doesn't yet support 'raise' annotations
```

Comment:
We should just remove those; I'm pretty sure mypy doesn't support them either, and I thought we had them because mypy uses them.

Reply:
Yes

Reply:
turns out it was just a syntax problem: pytype does support f(x) : [newline] raise, but not f(x): raise. i've added the newline and removed the file from the blacklist.